config/index.js

```diff
@@ -31,6 +31,8 @@ config.govInterval = process.env.GOV_INTERVAL || 864e5;
 // eslint-disable-next-line camelcase
 config.vaccineInterval = process.env.VACCINE_INTERVAL || 864e5;
 // eslint-disable-next-line camelcase
+config.vaccineCoverageInterval = process.env.VACCINE_COVERAGE_INTERVAL || 864e5;
```

Comment:
Just double check with the source to see how often they update their data, so we can coordinate scraping correctly

Reply:
They update 24 hourly.